src/components/Button/Button.jsx

```diff
@@ -1,42 +1,22 @@
-import PropTypes from 'prop-types';
-import classnames from 'classnames';
-import styles from './styles.module.scss';
+/* eslint-disable react/prop-types */
+import React from 'react';
+import classNames from 'classnames';
 
-export default function Button({
-  className,
-  outline,
-  onClick,
-  children,
-  ...rest
-}) {
+export const Button = ({ className, children, variant, ...props }) => {
   return (
     <button
       type="button"
-      className={classnames(
-        styles.button,
-        outline ? styles.outline : styles.full,
-        className
+      {...props}
+      className={classNames(
+        className,
```

Comment:
A propriedade `className` deve ser o último argumento se não for ela não vai sobrescrever outras classes e não vai fazer sentido tê-la